src/styles/blocks/container.scss

```diff
@@ -0,0 +1,27 @@
+.container {
+  --coloums-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--coloums-count), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
you can just change the `coloums-count` var in media

Reply:
@media (min-width: 488px) {
  .container {
    grid-template-columns: repeat(var(--columns-count), 200px);
  }
}

@media (min-width: 768px) {
  .container {
    --columns-count: 3;
  }
}

@media (min-width: 1024px) {
  .container {
    --columns-count: 4;
  }
}